sql/sqlbase/structured.proto

```diff
@@ -254,8 +254,7 @@ message TableDescriptor {
   optional uint32 format_version = 17 [(gogoproto.nullable) = false,
       (gogoproto.casttype) = "FormatVersion"];
 
-  // replaced by State.
-  optional bool OBSOLETE_Deleted = 18 [(gogoproto.nullable) = false];
+  reserved 18; // Deleted, replaced by State.
```

Comment:
it's actually replaced by `state` (lowercase). Also this use of "Deleted" is confusing because it's not clear if it refers to the history of this field, or its original name (the latter would be wrong, because its name was "deleted" (again lowercase)).

Perhaps just remove the comment entirely?